packages/x-charts/src/BarChart/checkClickEvent.test.tsx

```diff
@@ -29,14 +29,26 @@ const config = {
 describe('BarChart - click event', () => {
   const { render } = createRenderer();
 
+  // TODO: Remove beforeEach/afterEach after vitest becomes our main runner
```

Comment:
If you plan to move package by package, you could use a more search friendly message like `TODO vitest:` to make them easier to find at the end of the migration

Reply:
I'll find them regardless ;)